tests/test_data.py

```diff
@@ -4,11 +4,29 @@
 class TestUser(BaseModel):
     __test__ = False
 
+    id: int
     email: str
     password: str
 
 
+class TestStore(BaseModel):
```

Comment:
Why do we need to create these test schemas? why can't we use those from real code?